README.adoc

```diff
@@ -44,8 +44,10 @@ Smallrye JWT supports many properties which can be used to customize the token p
 |Property Name|Default|Description
 |smallrye.jwt.verify.algorithm|`RS256`|Signature algorithm. Set it to `ES256` to support the Elliptic Curve signature algorithm.
 |smallrye.jwt.verify.key-format|`ANY`|Set this property to a specific key format such as `PEM_KEY`, `PEM_CERTIFICATE`, `JWK` or `JWK_BASE64URL` to optimize the way the verification key is loaded.
-|smallrye.jwt.token.header|`Authorization`|Set this property if another header such as `Cookie` is used to pass the token.
-|smallrye.jwt.token.cookie|none|Name of the cookie containing a token. This property will be effective only if  `smallrye.jwt.token.header` is set to `Cookie`.
+|mp.jwt.token.header
+(deprecated) smallrye.jwt.token.header|`Authorization`|Set this property if another header such as `Cookie` is used to pass the token.
+|mp.jwt.token.cookie
+(deprecated) smallrye.jwt.token.cookie|`Bearer`|Name of the cookie containing a token. This property will be effective only if  `smallrye.jwt.token.header` is set to `Cookie`.
```

Comment:
Here lets instead have `This property is deprecated, use 'mp.jwt.... instead.'` at the end of the description part.

Reply:
Ok. Btw, we already have these in the docs, so we now have 2 locations with the configs. Can we get rid of these section in the README and point to the docs?

Reply:
Yeah, makes sense.